dev/dev.md

```diff
@@ -0,0 +1,235 @@
+
+# mm9
+java -jar ~/dxWDL-0.79.1.jar compile atac.wdl -project "ENCODE Uniform Processing Pipelines Azure" -extras dev/workflow_opts/docker.json -f -folder /ATAC-seq/workflows/$VER/mm9 -defaults dev/examples/dx_azure/template_mm9.json
+
+# test sample
+java -jar ~/dxWDL-0.79.1.jar compile atac.wdl -project "ENCODE Uniform Processing Pipelines Azure" -extras dev/workflow_opts/docker.json -f -folder /ATAC-seq/workflows/$VER/test_ENCSR356KRQ_subsampled -defaults dev/examples/dx_azure/ENCSR356KRQ_subsampled_dx_azure.json
+```
+
+## Building genome data (for mito-only index)
+
+```
+conda/build_genome_data.sh hg38 /mnt/lab_data/pipeline_genome_data/hg38
+conda/build_genome_data.sh hg38_chr19_chrM /mnt/lab_data/pipeline_genome_data/hg38_chr19_chrM
+conda/build_genome_data.sh mm10 /mnt/lab_data/pipeline_genome_data/mm10
+conda/build_genome_data.sh mm10_chr19_chrM /mnt/lab_data/pipeline_genome_data/mm10_chr19_chrM
+conda/build_genome_data.sh hg19 /mnt/lab_data/pipeline_genome_data/hg19
+conda/build_genome_data.sh mm9 /mnt/lab_data/pipeline_genome_data/mm9
+
+cd /mnt/lab_data/pipeline_genome_data
+
+find -name *.fasta -delete
+find -name *.fa -delete
+
+mv hg19/hg19.tsv hg19_klab.tsv
+mv hg38_chr19_chrM/hg38_chr19_chrM.tsv hg38_chr19_chrM_klab.tsv
+mv hg38/hg38.tsv hg38_klab.tsv
+mv mm10_chr19_chrM/mm10_chr19_chrM.tsv mm10_chr19_chrM_klab.tsv
+mv mm10/mm10.tsv mm10_klab.tsv 
+mv mm9/mm9.tsv mm9_klab.tsv
+
+DIR_SRC=/mnt/lab_data/pipeline_genome_data
+
+PLATFORM=caper
+DIR_TARGET=https://storage.googleapis.com/encode-pipeline-genome-data
+for G in "hg19" "mm9" "hg38" "mm10" "hg38_chr19_chrM" "mm10_chr19_chrM";
+do
+  TSV1=${G}_klab.tsv
+  TSV2=${G}_$PLATFORM.tsv
+  cp $TSV1 $TSV2
+  sed -i "s~$DIR_SRC~$DIR_TARGET~g" $TSV2
+done
+
+PLATFORM=gcp
+DIR_TARGET=gs://encode-pipeline-genome-data
+for G in "hg19" "mm9" "hg38" "mm10" "hg38_chr19_chrM" "mm10_chr19_chrM";
+do
+  TSV1=${G}_klab.tsv
+  TSV2=${G}_$PLATFORM.tsv
+  cp $TSV1 $TSV2
+  sed -i "s~$DIR_SRC~$DIR_TARGET~g" $TSV2
+  gsutil cp $DIR_SRC/$G/ataqc/* $DIR_TARGET/$G/ataqc/
+  gsutil cp $DIR_SRC/$G/*.chrM.fa.gz $DIR_TARGET/$G/
+  gsutil cp $DIR_SRC/$G/bowtie2_index/*.chrM.* $DIR_TARGET/$G/bowtie2_index/
+  gsutil cp $DIR_SRC/$G/bwa_index/*.chrM.* $DIR_TARGET/$G/bwa_index/
+done
+gsutil cp *_gcp.tsv $DIR_TARGET/
+gsutil cp *_caper.tsv $DIR_TARGET/
+
+PLATFORM=aws
+DIR_TARGET=s3://encode-pipeline-genome-data
+for G in "hg19" "mm9" "hg38" "mm10" "hg38_chr19_chrM" "mm10_chr19_chrM";
+do
+  TSV1=${G}_klab.tsv
+  TSV2=${G}_$PLATFORM.tsv
+  cp $TSV1 $TSV2
+  sed -i "s~$DIR_SRC~$DIR_TARGET~g" $TSV2
+  gsutil cp $DIR_SRC/$G/ataqc/* $DIR_TARGET/$G/ataqc/
+  gsutil cp $DIR_SRC/$G/*.chrM.fa.gz $DIR_TARGET/$G/
+  gsutil cp $DIR_SRC/$G/bowtie2_index/*.chrM.* $DIR_TARGET/$G/bowtie2_index/
+  gsutil cp $DIR_SRC/$G/bwa_index/*.chrM.* $DIR_TARGET/$G/bwa_index/
+done
+gsutil cp *_aws.tsv $DIR_TARGET/
+
+PLATFORM=sherlock
+PREFIX=leepc12@login.sherlock.stanford.edu
```

Comment:
May not want to put this in a public readme? Unless you're fine with it.

Reply:
Yep this is removed from dev now.